src/assets/translations/en-us.yaml

```diff
@@ -261,6 +261,17 @@ containerRuntime:
       label: containerd
       description: Namespaces for container images; use with nerdctl.
 
+pathManagement:
+  label: Configure PATH
+  tooltip: Choose how you'd like Rancher Desktop to look for Supporting Utilities.
+  options:
+    rcFiles:
+      label: Automatic
+      description: Rancher Desktop edits your shell profile for you. Restart any open shells for changes to take effect.
+    manual:
+      label: Manual
+      description: Rancher Desktop will not change your PATH configuration; use your favorite editor to add ~/.rd/bin to your path manually.
```

Comment:
I'm getting really nitpicky with this one, but a couple more ideas (sorry I didn't notice these in the mockups):

- It might be a good idea to make the tense the same in the descriptions for `Manual` and `Automatic`. See below for what I'd suggest.
- `use your favorite editor` is unnecessary IMO. Again, see the below.

I'd suggest changing `Rancher Desktop will not change your PATH configuration; use your favourite editor to add ~/.rd/bin to your path manually.` to `Rancher Desktop does not change your PATH configuration; add ~/.rd/bin to your path manually.`.

Reply:
I like it, thanks!